src/commands.rs

```diff
@@ -464,7 +474,18 @@ where
                 Ok(Response::CompileFinished(result)) => {
                     return handle_compile_finished(result, stdout, stderr)
                 }
-                Ok(_) => bail!("unexpected response from server"),
+                Ok(_) => {
```

Comment:
It is a programming error to see an unexpected response. I think this is not needed.